src/particles/elements/TaperedPL.H

```diff
@@ -0,0 +1,142 @@
+/* Copyright 2022-2023 The Regents of the University of California, through Lawrence
+ *           Berkeley National Laboratory (subject to receipt of any required
+ *           approvals from the U.S. Dept. of Energy). All rights reserved.
+ *
+ * This file is part of ImpactX.
+ *
+ * Authors: Chad Mitchell, Axel Huebl
+ * License: BSD-3-Clause-LBNL
+ */
+#ifndef IMPACTX_TAPEREDPL_H
+#define IMPACTX_TAPEREDPL_H
+
+#include "particles/ImpactXParticleContainer.H"
+#include "mixin/alignment.H"
+#include "mixin/beamoptic.H"
+#include "mixin/thin.H"
+#include "mixin/nofinalize.H"
+
+#include <AMReX_Extension.H>
+#include <AMReX_REAL.H>
+
+#include <cmath>
+
+
+namespace impactx
+{
+    struct TaperedPL
```

Comment:
Shall we call this `ThinTaperedPL`, anticipating that we will add a 2nd order thick element of this?

Reply:
See above: we will solve this in a follow-up for multiple elements, including multipoles.